G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs

```diff
@@ -403,6 +404,33 @@ public string GetTextValue(SeleniumCommandArguments search, TimeSpan timeout)
             return res;
         }
 
+        public DataTable GetTableElement(SeleniumCommandArguments search, TimeSpan timeout)
+        {
+            PreCheckCurrentWindowHandle();
+            if (!string.IsNullOrEmpty(search.IFrameSearch?.Value))
+                webDriver.SwitchTo().Frame(FindElement(search.IFrameSearch.Value, search.IFrameBy.Value, timeout));
+            var element = FindElement(search.Search.Value, search.By.Value, timeout);
+
+            if (element == null || element.TagName != "table")
+            {
+                throw new Exception("Cannot found the HTML table element. Try to change the search phrase so that the correct element is found");
```

Comment:
"Cannot found the HTML table element" => "Cannot find"/"Html table element not found" ;)

Reply:
thanks a lot! 😽